packages/cloudflare/src/utils/generate-routes-json.ts

```diff
@@ -186,7 +186,7 @@ export async function createRoutesFile(
 			const staticPath = staticFile;
 
 			const segments = removeLeadingForwardSlash(staticPath)
-				.split(posix.sep)
+				.split(process.platform === 'win32' ? win32.sep : posix.sep)
```

Comment:
Could we simplify this as:

```js
import path from 'node:path'

//...
.split(path.sep)
```

Reply:
Yes looks like this will work too. I'll adjust it on Monday.

Reply:
@alexanderniebuhr @bluwy done.
Can you run the workflow tests once more.